modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxLocalAdapter.java

```diff
@@ -1616,20 +1613,6 @@ public boolean queryEnlisted() {
         return qryEnlisted;
```

Comment:
Do we need fiels 'qryEnlisted' any more? Can we remove it and return false?

Reply:
May be, but this is not related to the PR explicitly.
Hope this will be done later.